content/news/release-20150425.md

```diff
@@ -0,0 +1,61 @@
+We are very pleased to announce the Spring release of OpenRA.  This is a huge step forward for the Dune 2000 mod in particular, but also, behind the scenes, for the still unreleased, work-in-progress Tiberian Sun mod which has received support for some crucial features during this cycle.
+
+This release includes just shy of 1000 commits by 53 contributors, the culmination of six months of relentless work, and with more than 2000 files and 210,000 changed lines one of the busiest development cycles to date.
+
+A large development focus has been on the Dune 2000 mod, which has made some huge steps forward and contains the most visible changes. A new sidebar, sandworms and carryalls are the big new features, which should make any Dune 2000 player feel more at home in our mod. Additionally, the OpenRA.Utility now supports importing maps from the original Dune 2000 game, so we hope that the number of maps and of course the number of players will increase in the near future!
+
+The other mods were not neglected, though. In Red Alert, the standout feature is the support of playable factions such as Russia, England, France, Ukraine and Germany. In both Red Alert and Tiberian Dawn, new single-player missions are available that will keep you occupied.
+
+Of course, we have not only improved the mods themselves. Here is a list of changes that affect the game in general:
+
+* The server browser shows and allows joining games from any installed mod.
+* It is now possible to disable the "short game" mode that has been the de-facto default in the past.
```

Comment:
Technically it was the de jure default (since it was forced by the game code).  How about something like "that was forced in earlier versions"?